### PR TITLE
Refactor test hsv2rgb

### PR DESCRIPTION
**Pull request by vivekkumar7089**
Refactor group B1 as mentioned in #3956 

Group B1
 test_hsv2rgb
 test_rgb2hsv
 test_rgb_to_grayscale -- parametrize over num_output_channels
 test_center_crop
 test_five_crop
 test_ten_crop

### CODE REVIEW

**Comment by NicolasHug** [comment]
Thanks @vivekkumar7089 ! I'll review in more details but it looks like something went wrong: I see no deleted tests, and it looks like you included `test_rotate` which was recently ported in another PR. Could you take a look at this?

**Comment by vivekkumar7089** [comment]
Thanks, @NicolasHug , for reviewing. I missed deleting the tests while removing merge conflicts. I will change as soon as possible.

**Review by NicolasHug — APPROVED** [review]
Nice! Thanks a lot @vivekkumar7089 !

File: test/test_functional_tensor.py [hunk 1]
```diff
@@ -36,160 +36,89 @@ class Tester(unittest.TestCase):
...
 
-        # scriptable function test
-        s_tuple_transformed_batches = script_ten_crop(batch_tensors, [10, 11])
-        for transformed_batch, s_transformed_batch in zip(tuple_transformed_batches, s_tuple_transformed_batches):
-            assert_equal(transformed_batch, s_transformed_batch)
+        # assert changed type warning
+        with self.assertWarnsRegex(UserWarning, r"Argument interpolation should be of type InterpolationMode"):
+            res1 = F.rotate(tensor, 45, interpolation=2)
+            res2 = F.rotate(tensor, 45, interpolation=BILINEAR)
+            assert_equal(res1, res2)
```
Comment (fmassa) [inline test/test_functional_tensor.py:121]: Is this still left-over from a merge conflict?

**Pull request merged**